src/core_ct/core.py

```diff
@@ -104,7 +104,7 @@ def slice(self, axis: int, loc: int) -> Slice:
             case _:
                 raise ValueError("axis must be a value between 0 and 2 (inclusive)")
 
-    def trim(self, axis: int, loc_start: int, loc_end: int | None = None) -> Core:
+    def trim(self, axis: int, left_index: int, right_index: int | None = None) -> Core:
```

Comment:
I'm not sure I'm sold on left and right, since you could be trimming along the z axis where top and bottom would be more applicable